### PR TITLE
fix/rewrite

### PR DESCRIPTION
**Pull request by Frzk**
This is a complete rewrite of the buildpack `bin/compile` file:
- allows to specify a Metabase version through the `METABASE_VERSION` environment variable
- when not specified or when set to `*` or `latest`, defaults to the latest released version available on Github
- caching support

The `README` file has been emptied for now.

### CODE REVIEW

**Review by Soulou — APPROVED** [review]
Waw great changes, congrats! If tested manually, LGTM

File: bin/version
Comment (yanjost) [inline bin/version:1]: @Frzk why not write the version requested in a file whose value could be returned by the "version" script ?

**Review by yanjost — COMMENTED** [review]
Excellent work. I like the namespacing of the bash functions with ::

File: bin/version
Comment (Frzk) [reply bin/version:1]: We have an environment variable to handle this :) (`METABASE_VERSION`)

Comment (yanjost) [reply bin/version:1]: Yes, but this doesn't log the actual version, only the desired version. If you forget to trigger a deployment, it won't be right, no ?

**Pull request merged**